schemas/context/media-timed.schema.json

```diff
@@ -109,6 +129,16 @@
           "title": "Total pause duration",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "Describes the duration in seconds in which playback was paused by the user."
+        },
+        "xdm:idp": {
+          "title": "MVPD Identifier",
+          "type": "string",
+          "description": "The MVPD provided via Adobe authentication."
+        },
+        "xdm:mediaAuth": {
+          "title": "Media authorized",
+          "type": "string",
+          "description": "The user has been authorized via Adobe authentication."
```

Comment:
This is only good for Adobe authentication??

Reply:
Yes, this is specific to our product and to Adobe authentication. The accepted values are not only true/false as they would be for a generic "Media authorized" field, it could also contain client custom data.

Reply:
Hmm...this is a problem since Adobe-specific stuff can't be part of a public schema.  Can you move these values to a separate schema, in the Adobe section, and then mix them in?

Reply:
Checked this again and it seems it's Adobe authentication for our product, but it's actually not Adobe specific. Corrected the description.